openapi/index.yaml

```diff
@@ -1426,50 +1426,54 @@ definitions:
         $ref: '#/definitions/MessageStatusValue'
     required:
       - message
-  RCConfiguration:
+  RCConfigurationResponse:
     allOf:
       - $ref: '#/definitions/RCConfigurationBase'
       - $ref: '#/definitions/RCConfigurationEnvironment'
   RCConfigurationBase:
-    type: object
-    properties:
-      userId:
-        type: string
-        minLength: 1
-        description: User id.
-      configurationId:
-        type: string
-        description: Ulid string.
-        format: Ulid
-        x-import: '@pagopa/ts-commons/lib/strings'
-        example: 01ARZ3NDEKTSV4RRFFQ69G5FAV
-      name:
-        type: string
-        minLength: 1
-        description: Configuration name.
-      description:
-        type: string
-        minLength: 1
-        description: Configuration description.
-      hasPrecondition:
-        $ref: '#/definitions/HasPrecondition'
-      disableLollipopFor:
-        type: array
-        items:
-          type: string
-          minLength: 1
-          description: User for which lollipop should be disabled
-      isLollipopEnabled:
-        type: boolean
-        description: Flag to check if lollipop is enabled
-    required:
-      - userId
-      - configurationId
-      - name
-      - description
-      - hasPrecondition
-      - disableLollipopFor
-      - isLollipopEnabled
+    allOf:
+      - type: object
+        properties:
+          name:
+            type: string
+            minLength: 1
+            description: Configuration name.
+          description:
+            type: string
+            minLength: 1
+            description: Configuration description.
+          hasPrecondition:
```

Comment:
we used camelCase in order to use only one spec both for the model and the dto (request response). Imho it was not worth creating two different specs just for the case

Reply:
finally, our codegen has an option to generate camelCased runtime types: https://github.com/pagopa/openapi-codegen-ts/blob/master/README.md?plain=1#L154

Reply:
Isn't this the same thing that is done with, for example, the MessageContentBase?
That definition is used both in the model and in the response, the only difference is that there the snake_case is used also in the model.

Reply:
We will modify the definitions in order to fix this problem. Anyway this is out of scope of this pr since the definitions comes from having updated io-functions-commons and are not used in the getMessage but in another api.

Reply:
I don't have enough context but if it is, I won't take that as a best practice. Moreover, at least in that case we maintain coherence in API specs.

Reply:
We managed to do the fix here [28f2e97](https://github.com/pagopa/io-functions-commons/pull/364/commits/28f2e9744bf84c9944a57cae527e3752e4b5ce43) cc @gunzip

Reply:
All the specs for the remote content configuration are now in snake_case as required cc @gunzip